src/components/navbar.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+import logo from '../images/yts-logo.png';
+import SearchBar from '../containers/search_bar';
```

Comment:
Instead of having to use relative paths everywhere, change project setup to accept absolute paths.
Example: import SearchBar from '../containers/search_bar'; => import SearchBar from 'containers/search_bar';
Than change the imports throughout the project.